cloudformation.yaml

```
@@ -117,10 +117,31 @@ Resources:
       TreatMissingData: notBreaching
       AlarmActions:
         - !Ref AlarmSnsTopic
-  LambdaLogErrorsMetricFilter:
+  # Alarm on WARN, ERROR and FATAL metrics.
+  # -ResourceLeakDetector to ensure we do not alarm on unavoidable and seemingly
+  # harmless error messages from Netty - see https://github.com/aws/aws-sdk-java-v2/issues/4654
+  LambdaLogWarnMetricFilter:
     Type: AWS::Logs::MetricFilter
     Properties:
-      FilterPattern: "?FATAL ?ERROR ?WARN"
+      FilterPattern: "WARN -ResourceLeakDetector"
+      LogGroupName: !Sub "/aws/lambda/${FunctionName}"
+      MetricTransformations:
+        - MetricValue: 1
+          MetricNamespace: content-api/podcasts-analytics-lambda
+          MetricName: LogErrors
+  LambdaLogErrorWarnMetricFilter:
```

Would it be clearer to only have `Error` in this filter name (leaving out `Warn`)?

```suggestion
  LambdaLogErrorMetricFilter:
```